lighthouse-cli/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "scripts": {
```

Comment:
def not postinstall since it does not do what you expect, maybe to prepublish though.. but this is more related to how we do releases which is already pretty complex. I would prefer to simply make a release script than shoehorn the functionality into the lifecycle scripts of npm.